lib/internal/Magento/Framework/View/Layout/etc/elements.xsd

```diff
@@ -14,6 +14,7 @@
                     <xs:sequence>
                         <xs:element name="updater" type="updaterType" minOccurs="0" maxOccurs="1"/>
                     </xs:sequence>
+                    <xs:attribute name="shared" type="xs:boolean" default="true" use="optional"/>
```

Comment:
In order to maintain backward compatibility please set this attribute to be `false` by default